tests/models/opt/test_modeling_opt.py

```diff
@@ -105,10 +105,10 @@ def prepare_config_and_inputs(self):
         )
         input_ids[:, -1] = self.eos_token_id  # Eos Token
 
-        decoder_input_ids = ids_tensor([self.batch_size, self.seq_length], self.vocab_size)
+        # decoder_input_ids = ids_tensor([self.batch_size, self.seq_length], self.vocab_size)
 
         config = self.get_config()
-        inputs_dict = prepare_opt_inputs_dict(config, input_ids, decoder_input_ids)
+        inputs_dict = prepare_opt_inputs_dict(config, input_ids)#, decoder_input_ids)
```

Comment:
Why are those commented out?

Reply:
They are not used in the `prepare_opt_inputs_dict` and should thus be deleted

Reply:
Let's remove them entirely then :-)